### ISSUE TITLE
Video is downloading in a different Audio Option

### ISSUE THREAD

**Issue by RedDragoon101** [shaked6540/YoutubePlaylistDownloader#408]
The video I am downloading has multiple Audio options, specifically Portuguese, English, and Spanish, and there isnt an option to choose which audio to use so its defaulting to Spanish, when I want it in English

**Comment by shaked6540** [comment]
Please add a link to the video. 
Also what do you mean its downloading audio in a different language? Did you mean the subtitles are downloading in a different language?

**Comment by RedDragoon101** [comment]
No, I mean YouTube has a built in feature that allows creators to have multiple different audio's for different languages, 
Like this video: https://www.youtube.com/watch?v=el_PChGfJN8&list=PLJSCtUUZr_nL-DJWXvHtvZQGemZe3dCbx&index=2
Or This Video: https://www.youtube.com/watch?v=4ofJpOEXrZs&t=759s

**Comment by RedDragoon101** [comment]
Its in the settings under Audio Track

**Comment by shaked6540** [comment]
Wow, didn't know about that one. I'll try to see what it takes to add it to the program, if its a small change I'll add it soon

**Comment by morghipala** [comment]
Is there a solution already?

**Comment by Josileissu** [comment]
Im waiting for this update...

**Comment by shaked6540** [comment]
I'm working on it, I hope it will be ready in the coming week :)

**Comment by Josileissu** [comment]
I coded in python language with ytb-dl an algorithm to download a multiple urls.

Here is:

import yt_dlp as youtube_dl
import os
from concurrent.futures import ThreadPoolExecutor

# Caminho do diretório onde os arquivos serão salvos
download_path = r"C:\videos"  # Substitua pelo caminho desejado

# Certifique-se de que o diretório existe
if not os.path.exists(download_path):
    os.makedirs(download_path)

# Lista de URLs dos vídeos do YouTube
urls = [
'urls1',
'url2'
]

# Configurações do yt-dlp para download do vídeo com áudio em português
ydl_opts = {
    'format': 'bestvideo+bestaudio/best',  # Baixar o melhor vídeo e áudio disponíveis
    'merge_output_format': 'mkv',  # Formato final do vídeo (pode ser mkv, mp4, etc)
    'subtitleslangs': ['pt'],  # Preferir legendas em português (caso existam)
    'writesubtitles': False,  # Baixar as legendas (se existirem)
    'writeautomaticsub': False,  # Não baixar legendas automáticas
    'postprocessors': [{
        'key': 'FFmpegMerger',  # Junta vídeo e áudio
    }],
    'prefer_ffmpeg': True,  # Usar ffmpeg para processamento
    'noplaylist': True,  # Desabilitar download de playlists
    'outtmpl': os.path.join(download_path, '%(title)s.%(ext)s'),  # Nome do arquivo de saída
}

# Função para baixar um único vídeo
def download_single_video(url):
    try:
        with youtube_dl.YoutubeDL(ydl_opts) as ydl:
            ydl.download([url])
        print(f"Download concluído: {url}")
    except Exception as e:
        print(f"Erro ao baixar {url}: {e}")

# Função principal para baixar vídeos simultaneamente
def download_videos_simultaneously(urls, max_threads=4):
    with ThreadPoolExecutor(max_threads) as executor:
        executor.map(download_single_video, urls)

# Chama a função para download simultâneo
if __name__ == "__main__":
    download_videos_simultaneously(urls)
